user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_Create_CCAP_CM_pair.md

```diff
@@ -4,42 +4,52 @@ uid: I-DOCSIS_Create_CCAP_CM_pair
 
 # Creating a new CCAP/CM pair
 
-To create CCAP/CM pairs, you can either start from scratch, or start from a CSV file.
+To create CCAP/CM pairs, you can either create a single CCAP/CM pair, or create elements in bulk from a CSV file.
 
-## Creating a new CCAP/CM pair from scratch
+## Creating a single CCAP/CM pair
 
 1. In DataMiner Cube, go to apps > Automation.
 
 1. In the pane on the left, select the script *EPM_I_DOCSIS_AddNewCcapCmPair*.
 
 1. In the lower right corner, click *Execute*.
 
-   This will open a wizard with several steps.
+   This will open a pop-up window.
+
+1. Click *Create single*.
 
 1. In the **General** step:
 
-   1. Define the element name.
+   1. Define the element name (mandatory field).
 
       > [!NOTE]
       > The name of the collector element will consist of the defined element name with the suffix "_COLLECTOR".
 
-   1. Select one of the back-end DMAs for the CCAP.
+   1. Select the host for the CCAP.
 
-   1. Select one of the back-end DMAs for the collector.
+   1. Select the host for the collector.
 
 1. In the **CCAP Details** step:
 
    1. Select the desired CCAP protocol.
 
    1. Select the desired protocol version. We recommend the production version.
 
-   1. Enter the IP address.
+   1. Enter the IP address (mandatory field).
 
-   1. Enter the community string that will be used to poll the device (SNMP).
+   1. Enter the get community string that will be used by the CCAP to poll the device (SNMP).
 
-1. In the **View Details** step, select the location where the CCAP/CM pair will be added in the EPM view structure.
+   1. Enter the set community string that will be used by the CCAP to set information in the device (SNMP).
+
+   1. Enter the get community string for the collector.
+
+   1. Enter the set community string  for the collector.
 
-1. In the **Interface Setting** step, define the username and password of an account that can access the **front-end DMA**.
+   1. Enter the user for this device.
+
+   1. Enter the password for this device.
+
+1. In the **View Details** step, select the location where the CCAP/CM pair will be added in the EPM view structure.
 
```

Comment:
Can we add a note indicating what is the purpose of this user/password?
If I am not mistaken, this is required to access a shared resource (shared folder in this case)
If so, can we mention if we support domain accounts? or only local accounts?

Reply:
@EstebanSkyline, could you confirm if these credentials are indeed used to access a shared folder, and if we support domain accounts or only local accounts?

Reply:
@MariekeGO Indeed, it is a mistake, it is not the password of the device, it is the password to access the shared folders.

Reply:
@EstebanSkyline, what is the purpose of these shared folders? Also, do we support domain accounts or only local accounts for the user?

Reply:
@MariekeGO Since the Collector element creates CSV files that the CCAP element reads, it is necessary to have shared folders credentials. And domain accounts are supported.